Adding a persistent scrollbar for media gallery to fix flickering.

ezxyz opened the pull request:
## Summary
Adding a persistent scrollbar to local media to stop the jittering. More info in #4311

<!-- A brief description of what this PR does. -->

## Relevant Technical Choices
Also removed logic around calculating padding size based on whether or not there is a scrollbar (because there will always be a scrollbar)


<!-- Please describe your changes. -->


## User-facing changes

There is now a scrollbar in the local gallery even when there is no overflow. (See the very faint scrollbar in this screenshot)
![Screen Shot 2020-09-08 at 4 10 42 PM](https://user-images.githubusercontent.com/13947816/92553836-5b184980-f2a7-11ea-9d4a-0d247107bd8f.png)

Fixes #4311

Commit:
persistent scrollbar

joannag6 commented on assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js:
```diff
@@ -61,33 +61,7 @@ function PaginatedMediaGallery({
   setNextPage,
 }) {
   // TODO(#1698): Ensure scrollbars auto-disappear in MacOS.
```
We can remove this TODO as well (That issue has also been closed)

ezxyz replied:
Done.

Commit:
Remove TODO(#1698): ...

Commit:
Merge branch 'main' into persistent-scrollbar

Commit:
Merge branch 'main' into persistent-scrollbar

Commit:
Merge branch 'main' into persistent-scrollbar

Commit:
Adding the refcallback again as it was being used to load on scroll.

(Accidentally removed it before thinking it was only being used to calculate the margin)

Commit:
Update paginatedMediaGallery.js

Commit:
Removed ref callback since using the ref will have the same behaviour

See https://reactjs.org/docs/refs-and-the-dom.html
